bunsen/bunsen-avro/src/test/java/com/cerner/bunsen/avro/R4AvroConverterCustomProfileTest.java

```diff
@@ -0,0 +1,363 @@
+package com.cerner.bunsen.avro;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.context.FhirVersionEnum;
+import com.cerner.bunsen.ProfileMapperFhirContexts;
+import com.cerner.bunsen.exception.ProfileMapperException;
+import com.cerner.bunsen.r4.TestData;
+import java.net.URISyntaxException;
+import java.util.List;
+import org.apache.avro.generic.GenericData.Record;
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.Extension;
+import org.hl7.fhir.r4.model.Patient;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class R4AvroConverterCustomProfileTest {
```

Comment:
nit: Consider adding a short comment to describe the general plan in these tests (e.g., which profiles are involved and how is it different from R4 and STU3 versions.

Reply:
Added comments, to clarify these are just some additional test cases for user defined profiles.